client/src/pages/Profile.js

```diff
@@ -0,0 +1,316 @@
+    flexGrow: "1",
+    alignItems: "center"
+  },
+  label: {
+    marginLeft: "15px",
+    font: theme.typography.fontFamily,
+    fontSize: ".6em",
+    fontWeight: 250,
+    letterSpacing: "0em",
+    color: "black",
+    whiteSpace: "nowrap"
+  },
+  logo: {
+    width: "30px"
+  },
+  dealsMateGrid: {
+    marginLeft: "20px"
+  },
+  dealsMateName: {
+    font: theme.typography.fontFamily,
+    fontSize: "1em",
+    fontWeight: 250,
+    letterSpacing: "0.5em",
+    color: "black"
+  },
+  badgeColor: {
+    color: theme.primary
+  },
+  accountCircle: {
+    color: theme.primary,
+    width: "1.5em",
+    height: "1.5em",
+    marginLeft: "40px"
+  },
+  profile: {
+    marginLeft: "5px",
+    font: theme.typography.fontFamily,
+    fontSize: ".6em",
+    fontWeight: 250,
+    letterSpacing: "0.1em",
+    color: "black",
+    whiteSpace: "nowrap"
+  },
+  bodyStart: {
+    paddingTop: "100px"
+  },
+  addNewItem: {
+    font: theme.typography.fontFamily,
+    fontSize: "2em",
+    fontWeight: 500,
+    letterSpacing: "0em",
+    color: "black"
+  },
+  paperLinkContainer: {
+    borderRadius: "50px 0px 0px 50px",
+    boxShadow: "none",
+    borderRight: "1px solid WhiteSmoke"
+  },
+  linkInput: {
+    marginLeft: 25,
+    marginTop: 10,
+    marginBottom: 10,
+    width: "25vw",
+    fontSize: ".8em"
+  },
+  paperListContainer: {
+    borderRadius: "0px 50px 50px 0px",
+    boxShadow: "none",
+    display: "flex",
+    flexDirection: "row",
+    alignItems: "center"
+  },
+  emptyListSelect: {
+    fontSize: ".8em",
+    color: "grey"
+  },
+  nonEmptyListSelect: {
+    fontSize: ".8em"
+  },
+  listSelect: {
+    marginLeft: 25,
+    marginTop: 10,
+    marginBottom: 10,
+    width: "8vw"
+  },
+  addItemButton: {
+    marginLeft: 25,
+    marginRight: 10,
+    backgroundColor: theme.primary,
+    fontSize: ".8em",
+    color: "white"
+  },
+  shoppingListsContainer: {
+    marginLeft: "10vw",
+    marginRight: "10vw",
+    marginTop: "10vh"
+  },
+  shoppingListText: {
+    font: theme.typography.fontFamily,
+    fontSize: "2em",
+    fontWeight: 500,
+    letterSpacing: "0em",
+    color: "black"
+  },
+  listTile: {
+    backgroundColor: "white",
+    marginRight: "20px",
+    borderRadius: "15px",
+    marginTop: "20px"
+  }
+}));
+
+const StyledBadge = withStyles(theme => ({
+  badge: {
+    top: "25%",
+    right: 5,
+    background: `${theme.primary}`
+  }
+}))(Badge);
+
+function ProfilePage() {
+  const [list, setList] = useState("");
+
+  const handleChange = event => {
+    setList(event.target.value);
+  };
+
+  const listOptions = ["a", "b", "c"];
+  const listNames = ["Clothes", "Furniture", "Luxury"];
+
+  const classes = profilePageStyles();
+
+  function getClassName() {
+    if (list === "") {
+      return classes.emptyListSelect;
+    } else {
+      return classes.nonEmptyListSelect;
+    }
+  }
+  return (
+    <div className={classes.backgroundColor}>
+      <AppBar className={classes.topBar}>
```

Comment:
Did you see my request in the other PR - it would be great to put this actually at a higher level component (in the App, and make this a component itself) so it could exist in all the pages easily